dll/win32/browseui/shellfind/CSearchBar.cpp

```diff
@@ -32,6 +32,31 @@ CSearchBar::~CSearchBar()
 
 LRESULT CSearchBar::OnInitDialog(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHandled)
 {
+    HKEY hkey;
+    DWORD dwType = REG_DWORD; 
+    DWORD size = sizeof(DWORD);
+    DWORD regOpenResult;
+    DWORD SearchHiddenvalue = 0;
+
+    regOpenResult = RegOpenKeyEx(HKEY_CURRENT_USER, L"Software\\Microsoft\\Windows\\CurrentVersion\\Explorer", 0, KEY_ALL_ACCESS, &hkey);
+    if(regOpenResult == ERROR_SUCCESS)
+    {
+        if (RegQueryValueEx(hkey, L"SearchHidden", NULL, &dwType, (LPBYTE)&SearchHiddenvalue, &size) == ERROR_SUCCESS)
+        {
+            ERR("SearchHidden is '%d'.\n", SearchHiddenvalue);
+            RegCloseKey(hkey);
```

Comment:
This line must be outside of this sub-block.

Reply:
OK.